Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# One Blind Eye

## Changelog:
- If you have the One-Eyed trait, your left eye is gray blind and you have a scar.
- Only humans wear eye patches if they are one-eyed.


# How to test:
- `add_trait one_eyed` in the console to add the trait.
Check how the One-Eyed affects characters of different cultures.
